src/utils/ValidateConfigs.ts

```diff
@@ -3,55 +3,62 @@ import { GoogleMapContainerProps } from "../components/GoogleMapContainer";
 export class ValidateConfigs {
 
     public static validate(props: GoogleMapContainerProps): string {
-        let message = "";
+        const message: string[] = [];
         const invalidEnumKeys = props.markerImages.filter(markerImage =>
-            /\s/.test(markerImage.enumKey)
+            /^\w+$/.test(markerImage.enumKey)
         );
```

Comment:
yes, you check for invalid keys, though it does not make much sens....
The check should happen at runtime and be against domain model 
https://apidocs.mendix.com/7/client/mendix_lib_MxMetaObject.html#getOptions

Reply:
@jseremba to remove this check. it is not necessary